PublishParasoftTestResults/publishtestresults.ts

```diff
@@ -65,11 +70,28 @@ if (isNullOrWhitespace(searchFolder)) {
     searchFolder = tl.getVariable('System.DefaultWorkingDirectory');
 }
 
+let dtpBaseUrl : string;
+let dtpUsername : string;
+let dtpPassword : string;
 const localSettings = loadSettings(localSettingsPath);
 if (localSettings) {
     tl.debug('dtp.url: ' + localSettings['dtp.url']);
     tl.debug('dtp.user: ' + localSettings['dtp.user']);
     tl.debug('dtp.password: ' + localSettings['dtp.password']);
+    tl.debug('dtp.server: ' + localSettings['dtp.server']);
```

Comment:
Let's only print out these debug message when there's valid value for each property.